deploy/lib/deployer/repo.rb

```diff
@@ -1,3 +1,11 @@
+require "dependabot/source"
+require "dependabot/file_fetchers"
+require "dependabot/file_parsers"
+require "dependabot/file_updaters"
+require "dependabot/pull_request_creator"
+require "dependabot/update_checkers"
+require "dependabot/npm_and_yarn"
+
```

Comment:
I'm curious, why are we `require`ing the Dependabot code here and not in the `DependabotPullRequestUpdater` class?

Reply:
Good catch!  I originally was writing the code in this class, but then moved it out but missed these requires!  I'll move them.